mesonbuild/compilers/compilers.py

```diff
@@ -1619,6 +1621,27 @@ def get_no_warn_args(self):
         def openmp_flags(self):
             return ['-fopenmp']
 
+
+class FlangCompiler:
```

Comment:
Since the flang documentation says that flang accepts all args that clang does, plus a few additional fortran specific arguments, does it it make sense for this to inherit from `ClangCompiler`?

Reply:
done in c27d25c